xml/cl.xml

```diff
@@ -6338,6 +6400,7 @@ server's OpenCL/api-docs repository.
                 <type name="cl_svm_mem_flags_arm"/>
                 <type name="cl_kernel_exec_info_arm"/>
                 <type name="cl_device_svm_capabilities_arm"/>
+                <type name="cl_free_svm_callback"/>
```

Comment:
Same here, this would need guards so it is only defined when cl.h does not provide it. Redefinition of typedefs is illegal in c99.